org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/GridTabRowRenderer.java

```diff
@@ -813,7 +823,7 @@ else if (!div.getChildren().isEmpty()) {
 		            Properties ctx = isDetailPane() ? new GridRowCtx(Env.getCtx(), gridTab) 
 		            	: gridPanelFields[i].getVO().ctx;
 		            //check context
-					if (!gridPanelFields[i].isDisplayed(ctx, true)){
+					if (! (gridPanelFields[i].isDisplayed(ctx, true) || gridPanelFields[i].isDisplayedGrid(ctx, true))){
```

Comment:
don't think it is right to call both here. should be isDisplayedGrid(ctx, true) for isDetailPane() and isDisplayed(ctx, true) otherwise.

Reply:
Checking IsDisplayedGrid flag, so...